app/src/main/res/values/strings.xml

```diff
@@ -690,5 +690,13 @@
   <string name="sheet_medicine_frequency_qds">QDS</string>
   <string name="sheet_medicine_frequency_save">Save</string>
 
-
-</resources>
+  <!-- Teleconsult Record Screen -->
+  <string name="screen_teleconsult_record">TELECONSULT RECORD</string>
```

Comment:
If we want this to be displayed in all caps, can we add this as an attribute to the `TextView` with `textAllCaps=true` instead of in the string resource?

Reply:
That's something I found out today. But I still feel we should use the attribute to control this because it's a property of the view and not of the string resource itself.

Reply:
This is okay for now, we can change it in the final PR.